src/components/SidebarButton.test.js

```diff
@@ -6,7 +6,7 @@ afterEach(cleanup);
 
 describe('<SidebarButton/>', () => {
   it('renders without crashing', () => {
-    global.renderWithRedux(<SidebarButton />);
+    global.renderWithRedux(<SidebarButton icon="icon" />);
```

Comment:
What does this addition do?

Reply:
removed, no longer needed with default props added